src/controllers/AccommodationController.js

```diff
@@ -0,0 +1,30 @@
+import dotenv from 'dotenv';
+import Response from '../helpers/Response';
+import AccommodationService from '../services/AccommodationService';
+
+
+dotenv.config();
+/**
+ * @exports
+ * @class AccommodationController
+ */
+class AccommodationController {
+  /**
+   * Travel Admin can be able to
```

Comment:
`Travel Admin can be able to...`
I think you forgot to complete it.
 I think it should be `Travel Admin can be able to add accommodation facility`